pkg/commands/edit/remove/all.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remove
+
+import (
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/pkg/fs"
+)
+
+// NewCmdRemove returns an instance of 'remove' subcommand.
+func NewCmdRemove(fsys fs.FileSystem) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "remove",
+		Short: "Removes items to the kustomization file.",
+		Long:  "",
```

Comment:
Hi. After a quick read of the tests, is `remove`, removing items `from` the kustomization file?
See lines 28 and 38.